internal/pkg/service/stream/keboolasink/token.go

```diff
@@ -0,0 +1,18 @@
+package keboolasink
+
+import (
+	"github.com/keboola/go-client/pkg/keboola"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/definition/key"
+)
+
+// Token is a Storage API Token used to create staging files and import files to the target table.
+// The token is scoped only to the target table, so there is a separate token for each definition.TableSink.
+type Token struct {
+	key.SinkKey
+	Token keboola.Token `json:"token"`
+}
```

Comment:
Isn't there better naming than `Token.Token`?  For example `Token.StorageToken`.

Reply:
:man_shrugging: In Keboola there is:
- Storage API token
- Management API Token

The `go-client` and all in KAC repo is using only Storage API token, but it may change in the future. So I would not deal with it now and rename everywhere if necessary, in the future.